inc/signalwire-client-c/connection.h

```diff
@@ -120,7 +120,6 @@ SWCLT_DECLARE(ks_status_t) swclt_conn_connect_ex(
 
 SWCLT_DECLARE(ks_status_t) swclt_conn_submit_request(swclt_conn_t *conn, swclt_cmd_t **cmd, swclt_cmd_future_t **future);
 SWCLT_DECLARE(ks_status_t) swclt_conn_submit_result(swclt_conn_t *conn, swclt_cmd_t *cmd);
-SWCLT_DECLARE(ks_status_t) swclt_conn_info(swclt_conn_t *conn, swclt_conn_info_t *info);
```

Comment:
can you put this back into connection?  Was this what was breaking the windows build?